plugins/autogrow/plugin.js

```diff
@@ -147,7 +147,13 @@
 			// to the one set by previous resizeEditor() call.
 			if ( newHeight != currentHeight && lastHeight != newHeight ) {
 				newHeight = editor.fire( 'autoGrow', { currentHeight: currentHeight, newHeight: newHeight } ).newHeight;
-				editor.resize( editor.container.getStyle( 'width' ), newHeight, true );
+
+				var boxSizingType = editor.container.getComputedStyle( 'box-sizing' );
+				var isBorderBox = boxSizingType === 'border-box';
+
+				var width = editor.container.getSize( 'width', isBorderBox );
```

Comment:
We are declaring variables using a single `var` statement when possible. Please, join variables.